<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据分析客户端 - 登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}
QMessageBox {
    background-color: white;
}
QMessageBox QPushButton {
    padding: 5px 15px;
    border-radius: 4px;
    background-color: #3a3f51;
    color: white;
    min-width: 80px;
    min-height: 25px;
}
QMessageBox QPushButton:hover {
    background-color: #2c3143;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="main_layout">
    <property name="spacing">
     <number>25</number>
    </property>
    <property name="leftMargin">
     <number>60</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>60</number>
    </property>
    <property name="bottomMargin">
     <number>40</number>
    </property>
    <item>
     <widget class="QLabel" name="title_label">
      <property name="text">
       <string>知微客户端</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 28px; font-weight: bold; color: #333; margin-bottom: 20px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="username_layout">
      <item>
       <widget class="QLabel" name="username_label">
        <property name="text">
         <string>用户名:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #333; font-size: 16px; margin-bottom: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username_edit">
        <property name="minimumHeight">
         <number>45</number>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    padding: 12px;
    border: 1px solid #ddd;
    border-radius: 6px;
    font-size: 16px;
    background-color: white;
    color: #000000;
}
QLineEdit:focus {
    border: 2px solid #3a3f51;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="password_layout">
      <item>
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>密码:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #333; font-size: 16px; margin-bottom: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_edit">
        <property name="minimumHeight">
         <number>45</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    padding: 12px;
    border: 1px solid #ddd;
    border-radius: 6px;
    font-size: 16px;
    background-color: white;
    color: #000000;
}
QLineEdit:focus {
    border: 2px solid #3a3f51;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="minimumHeight">
       <number>50</number>
      </property>
      <property name="text">
       <string>登 录</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #3a3f51;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 12px 24px;
    font-size: 18px;
    font-weight: bold;
    min-width: 250px;
    margin-top: 20px;
    letter-spacing: 4px;
}
QPushButton:hover {
    background-color: #2c3143;
}
QPushButton:pressed {
    background-color: #1f2532;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 